frontend/src/utils/projects.js

```diff
@@ -14,21 +14,9 @@
 // limitations under the License.
```

Comment:
do we want to keep this file? Only one method is remaining... we could also move the method to utils/index.js

Reply:
I will move it